docker-compose.yml

```diff
@@ -8,8 +8,8 @@ services:
       - "22300:22300"
     volumes:
       - ${PWD}/hackathons_canada/local_settings.py:/app/hackathons_canada/local_settings.py:ro
-      - /srv/static:/app/static2
-      - /srv/media:/app/media
+      - static_volume:/static
```

Comment:
caddy relies on these files being located at /srv/*

Reply:
Oh. I needed to use that in order for static files to properly load when running the web page from the docker container. I'll try not to push that again.